cypress/integration/3.IDIR-only-GOLD-no-Teams.feature

```diff
@@ -0,0 +1,180 @@
+        When User clicks on "+ Request SSO Integration" button
+        #Below is the reusable line for inputting text box value by passing the textbox label and value
+        And User Enter "CSS-HappyPath-end2end_Gold" on "Project Name" textbox
+        #Below is the reusable line for selecting radio button by passing text for radio button and value for it
+        And User select "No" for question "Would you like to allow multiple members to manage this integration?" on page
+        And User select "Yes" for legend question "Are you the product owner or technical contact for this project?" on page
+        And User hover over info button for text "Are you the product owner or technical contact for this project?" and verify the text "Only the person who is responsible for this project should be submitting the integration request."
+        And User clicks on "Next" button
+        And User select "Public" for legend question "Select Client Type" on page
+        And User select "IDIR" for legend question "Choose Identity Provider(s)" on page
+        And User select "Test" for legend question "Choose Environment(s)" on page
+        And User select "Production" for legend question "Choose Environment(s)" on page
+        And User clicks on "Next" button
+        And User Enters "https://localhost:3000" on "Redirect URIs" legend textbox at position "1"
+        And User hover over info button for text "Keycloak Login Page Name" and verify the text "Enter a name that you would like to be displayed for users, as they're logging into the Keycloak Login Page."
+        And User hover over info button for text "Redirect URIs" and verify the text "At least one redirect URI is required for each of DEV, TEST and PROD. If you don't know the redirect URI for one or"
+        And User clicks on "Next" button
+        And User Enters "https://localhost:3000" on "Redirect URIs" legend textbox at position "1"
+        And User clicks on "Next" button
+        And User Enters "https://localhost:3000" on "Redirect URIs" legend textbox at position "1"
+        And User clicks on "Next" button
+        #Below is reusableline for checkbox by passing value to select
+        And User Checks "I agree to the Terms and Conditions"
+        And User clicks on "Next" button
+        #Below is reusable line for validation by passing values
+        Then User validates information "Project Name" value is "CSS-HappyPath-end2end_Gold"
+        And User validates information "Identity Providers Required" value is "idir"
+        And User validates information "Dev Redirect URIs" value is "https://localhost:3000"
+        And User clicks on "Submit" button
+        And User clicks on "Confirm" button
+        And User Verfiy "Status" is "Submitted" in Table
+        And User Verfiy "Service Type" is "Gold" in Table
+        And User waits for "20" minutes
+        And User clicks on "Download" button at position "1"
```

Comment:
I believe the most benefits of BDD is the readibility and `position "1"` does not explain anything to me.